docs/index.rst

```diff
@@ -78,7 +78,11 @@ The asset rules are an array of JSON requires **rules** as an array of objects w
        * - Nested 
          - DP_DICT
 
-    Note: Datapoint types are case insensitive.
+         .. note:
```

Comment:
colon is missing i.e `note::`

Fixed in https://github.com/fledge-iot/fledge-filter-asset/pull/51